_api-reference/document-apis/index-document.md

```diff
@@ -33,6 +33,53 @@ PUT <index>/_create/<_id>
 POST <index>/_create/<_id>
 ```
 
+- PUT adds or updates documents with a specified ID in the index. Used for controlled document creation or updates.
+- POST adds documents to the index with auto-generated IDs. Useful for adding new documents without specifying IDs.
+- `_create` is a type identifier indicating document creation should only occur if the document with the specified ID doesn't already exist.
+- `<index>` represents the name of the index where the document will be added.
+- `<_id>` represents the unique identifier of the document.
+
+## Adding a sample index and documents to use the HTTP methods 
+
+To add a documents, do the following:
```

Comment:
```suggestion
If you want to quickly test the Document APIs, you can quickly add a document by doing the following:
```

Reply:
You probably don't want to use quickly twice in a sentence.

Reply:
Thank you @hdhalter this is now updated.